tests/foreman/api/test_registration.py

```diff
@@ -89,8 +88,10 @@ def test_host_registration_end_to_end(
 
 
 @pytest.mark.tier3
+@pytest.mark.rhel_ver_match('[^6]')
+@pytest.mark.skip_if_open("BZ:2229112")
```

Comment:
Can we keep the workaround of this BZ to register twice with condition of `is_open('BZ:2229112')` instead of skipping a test, wdyt?

Reply:
We should skip this until the issue gets fixed. It would be easier to fix I guess.